src/app/modules/wallet/pages/account-voluntary-exit/account-voluntary-exit.component.html

```diff
@@ -0,0 +1,54 @@
+<app-breadcrumb></app-breadcrumb>
+<div class="md:w-2/3">
+    <form [formGroup]="exitAccountFormGroup" (ngSubmit)="confirmation()">
+        <mat-card class="bg-paper">
+            <mat-card-content>
+                <h3 class="text-accent">
+                    Are you sure you want exit these account?
+
+                    Once these accounts are exited, the action cannot be
+                    reverted.
+                </h3>
+                <mat-selection-list (selectionChange)="selectionChange($event)"
+                    #shoes>
+                    <mat-list-option
+                        [value]="item.validatingPublicKey|base64tohex"
+                        *ngFor="let item of keys">
+                        {{item.accountName}}
+                    </mat-list-option>
+                </mat-selection-list>
+                <mat-form-field class="w-100">
+                    <mat-label>Confirmation</mat-label>
+                    <input matInput formControlName="confirmation" />
+                    <mat-hint>
+                        <h5>
+                            Type <i>'agree'</i> if you want to exit the keys
+                        </h5>
+                    </mat-hint>
+                    <mat-error>
+                        <span
+                            *ngIf="exitAccountFormGroup?.get('confirmation')?.hasError('required')">
+                            The confirmation is required</span>
+
+                        <span
+                            *ngIf="exitAccountFormGroup?.get('confirmation')?.hasError('incorectValue')">
+                            You must type agree
```

Comment:
Let's also have `'agree'` here for better readability. Also, let's merge both spans into one saying
> Confirmation is required. You must type 'agree'
Having two spans doesn't look well on https://deploy-preview-164--prysm-web-ui.netlify.app/dashboard/wallet/accounts/voluntary-exit

Reply:
I will fix this